app/views/admin/payments/index.html.haml

```diff
@@ -0,0 +1,27 @@
+%section.admin_payments
+  .container
+    .row
+      .col-sm-10.col-sm-offset-1
+        %section.payment_history
+          %ul.nav.nav-tabs
+            %li.active
+              %a(href="") Recent Payments
```

Comment:
If you don't want it to link anywhere the standard is normally to put something like `href="#"`.